automation/include/balena-api.inc

```diff
@@ -354,3 +364,323 @@ balena_api_is_dt_private() {
 	fi
 	echo "${_is_private}"
 }
+
+# Gets a release
+# Arguments:
+#
+# $1: Application name
+# $2: Release version string
```

Comment:
Can we clarify the syntax of the version string in this comment?

Reply:
It's going to depend on the app, but I guess I could say that it's usually a semver.